detect/fixeddetector.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+from .detector import Detector
+from dataset import Table
+
+class FixedDetector(Detector):
```

Comment:
`FixedDetector` does not sound very expressive. How about `ErrorsLoaderDetector`?

Reply:
Done.